templates/app-service-windows.json

```diff
@@ -39,6 +39,10 @@
         "description": "This can be passed into the template via the reference function: [reference(resourceId(parameters('certificateResourceGroup'), 'Microsoft.Web/certificates', parameters('certificateName')), '2016-03-01').Thumbprint]"
       }
     },
+    "appServiceState": {
+      "type": "string",
+      "defaultValue": "Running"
```

Comment:
I think it'd be worth putting allowedValues in there as well just to validate the input.

Reply:
Just added allowedValues and metadata as suggested.